src/ci/citool/src/metrics.rs

```diff
@@ -74,6 +74,17 @@ Maybe it was newly added?"#,
 }
 
 pub fn download_job_metrics(job_name: &str, sha: &str) -> anyhow::Result<JsonRoot> {
+    // Best effort cache to speed-up local re-executions of citool
+    let cache_path = PathBuf::from(".citool-cache").join(sha).join(format!("{job_name}.json"));
```

Comment:
shouldn't we also use `sha` for the cache?

Reply:
It is there :) `.join(sha)`. The `sha` determines the directory name, and the job name determines the file name.